src/app.ts

```diff
@@ -1,31 +1,33 @@
-const path = require('path')
-const {
+import { join as joinPath } from 'path'
```

Comment:
I'd like to keep having the default import `path` (or Node APIs in general) instead of using named import, because seeing `joinPath` could lead to thinking it's a custom function and not `path.join`.

Reply:
Fixed.  I was thinking along the lines of webpack tree-shaking but that concept isn't in place here.  Mixing up desktop and web apps :)